api_app/script_analyzers/file_analyzers/unpac_me.py

```diff
@@ -0,0 +1,99 @@
+import requests
+import logging
+from api_app.script_analyzers.classes import FileAnalyzer
+from api_app.exceptions import AnalyzerRunException
+import time
+from intel_owl import secrets
+from typing import Dict
+
+logger = logging.getLogger(__name__)
+
+
+class UnpacMe(FileAnalyzer):
+    base_url: str = "https://api.unpac.me/api/v1/"
+
+    def set_config(self, additional_config_params):
+        self.api_key_name = additional_config_params.get(
+            "api_key_name", "UNPAC_ME_API_KEY"
+        )
+        private = additional_config_params.get("private", False)
+        self.private = "private" if private else "public"
+        self.__api_key = secrets.get_secret(self.api_key_name)
+        # max no. of tries when polling for result
+        self.max_tries = additional_config_params.get("max_tries", 30)
+        # interval b/w HTTP requests when polling
+        self.poll_distance = 5
+
+    def run(self):
+        if not self.__api_key:
+            raise AnalyzerRunException(
+                f"No API key retrieved with name: {self.api_key_name}"
+            )
+        self.headers = {"Authorization": "Key %s" % self.__api_key}
+        unpac_id = self._upload()
+        logger.info(f"md5 {self.md5} job {self.job_id} uploaded id {unpac_id}")
+        for chance in range(self.max_tries):
+            time.sleep(self.poll_distance)
+            logger.info(
+                f"unpacme polling, try n.{chance + 1}."
+                f" job_id {self.job_id}. starting the query"
+            )
+            status = self._get_status(unpac_id)
+            if status == "fail":
+                logger.error(f"md5 {self.md5} job {self.job_id} analysis has failed")
```

Comment:
redundant `logger.error` line. Same error is already logged inside AnalyzerRunException.

Reply:
Yeah, you are right